resources/leiningen/new/luminus/core/src/nrepl.clj

```diff
@@ -1,22 +1,44 @@
 (ns <<project-ns>>.nrepl
   (:require
     [nrepl.server :as nrepl]
+    [clojure.java.io :as io]
```

Comment:
I would make the change for nrepl.clj cider conditional as it's not needed otherwise, and ends up adding some noise.